build.gradle

```diff
@@ -14,9 +17,61 @@ repositories {
 
 dependencies {
     api("org.openmicroscopy:omero-blitz:5.5.0-m7")
+    api("ome:formats-api:5.9.2") {
+        exclude group: "org.perf4j", module: "perf4j"
+    }
     implementation("commons-beanutils:commons-beanutils:1.9.3")
     implementation("com.zeroc:icegrid:3.6.4")
     testCompile("org.testng:testng:6.9.8")
+    // Re-activate from transitive excludes
+    //implementation("org.apache.commons:commons-collections4:4.1")
+    implementation("commons-collections:commons-collections:3.2.2")
+    implementation("commons-lang:commons-lang:2.6")
+    implementation("com.google.guava:guava:17.0")
+}
+
+configurations.all {
+    // resolutionStrategy.cacheChangingModulesFor 0, 'minutes'
+    exclude group: 'OME'
+    exclude group: 'antlr'
+    exclude group: 'asm'
+    exclude group: 'backport-util-concurrent'
+    exclude group: 'batik'
+    exclude group: 'cglib'
+    exclude group: 'com.codahale.metrics'
+    exclude group: 'com.drewnoakes'
+    exclude group: 'com.esotericsoftware.kryo'
+    exclude group: 'com.jamonapi'
+    exclude group: 'com.mortennobel'
+    exclude group: 'commons-codec'
+    exclude group: 'commons-pool'
+    exclude group: 'dom4j'
+    exclude group: 'edu.ucar'
```

Comment:
client using BF like insight will have to re-inject that in order for BF to work properly

Reply:
So far In order to build insight I had to add:
```
implementation("commons-io:commons-io:2.6")
    implementation("net.sf.ehcache:ehcache-core:2.6.11")
    implementation("ome:formats-bsd:5.9.2")
    implementation("commons-lang:commons-lang:2.6")
```